Formula/python-typing-extensions.rb

```diff
@@ -0,0 +1,53 @@
+class PythonTypingExtensions < Formula
+  desc "Backported and experimental type hints for Python"
+  homepage "https://github.com/python/typing_extensions"
+  url "https://files.pythonhosted.org/packages/9e/1d/d128169ff58c501059330f1ad96ed62b79114a2eb30b8238af63a2e27f70/typing_extensions-4.3.0.tar.gz"
+  sha256 "e6d2677a32f47fc7eb2795db1dd15c1f34eff616bcaf2cfb5e997f854fa1c4a6"
+  license "Python-2.0"
+
+  depends_on "flit" => :build
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "python@3.8" => [:build, :test]
+  depends_on "python@3.9" => [:build, :test]
+
+  def pythons
+    deps.select { |dep| dep.name.start_with?("python") }
+        .map(&:to_formula)
+        .sort_by(&:version)
+  end
+
+  def install
+    system Formula["flit"].opt_bin/"flit", "build", "--format", "wheel"
+    wheel = Pathname.glob("dist/typing_extensions-*.whl").first
+    pip_flags = %W[
+      -v
+      --no-deps
```

Comment:
```suggestion
      --verbose
      --isolated
      --no-deps
```
Maybe? We definitely want to spell out `-v` as `--verbose` for ease of reading, anyway.